internal/core/subsume/vertex.go

```diff
@@ -373,11 +381,18 @@ func (s *subsumer) verticesDev(x, y *adt.Vertex) bool {
 			continue
 		}
 
-		s.missing = f
-		s.gt = a
-		s.lt = y
+		if b == nil {
```

Comment:
ditto

Reply:
You are correct. I have removed this and also set the `s.missing` et al fields inside the block above.